src/annotator/guest.js

```diff
@@ -240,6 +240,17 @@ export default class Guest {
     this._listeners.add(window, 'resize', () => this._repositionAdder());
   }
 
+  /**
+   * Inject the Hypothesis client into a guest frame.
+   *
+   * @param {HTMLIFrameElement} frame
+   */
+  // eslint-disable-next-line no-unused-vars
+  async injectClient(frame) {
```

Comment:
This method has been added to the `Annotator` type. It would be helpful to add `@implements Annotator` to the `Guest` class to provide the conceptual glue here. Realizing that `Annotator` is an API that represents a subset of `Guest` took a few brain cycles :).